Resources/views/backend/swag_backend_order/controller/main.js

```diff
@@ -617,6 +620,12 @@ Ext.define('Shopware.apps.SwagBackendOrder.controller.Main', {
 
                 me.validationMail = pluginConfigObj.data.validationMail;
                 me.desktopTypes = pluginConfigObj.data.desktopTypes;
+                me.sendMail = pluginConfigObj.data.sendMail;
+                if (me.sendMail == 1) {
+                    me.orderModel.set('sendMail', 1);
+                    //Set checked class for table
+                    document.getElementById('sendMailChk').classList.add('x-form-cb-checked');
```

Comment:
You are totally right. I'll play more around with that tomorrow. I had no idea how to get the checkbox object in extjs in this method. That was just a workaround. Tomorrow I'll try to use the following selector with your mentioned method 
Ext.get("sendMailChk").setValue(true)